src/app/components/MessageBanner/README.md

```diff
@@ -0,0 +1,36 @@
+# MessageBanner
+
+## Description
+
+This component renders a message banner which comprises an H2 heading, a link (styled as a CTA), an image and an optional description as a paragraph. These elements sit within a region landmark. The banner can be rendered anywhere on a page and uses the curation `visualStyle` of `BANNER`. The banner background is generated programmatically with CSS radial and linear colour gradients at each breakpoint.
```

Comment:
I think we need a high level overview of the component, as well as this detailed description.